centaur/src/main/resources/standardTestCases/papi_cpu_platform/papi_cpu_platform.wdl

```diff
@@ -0,0 +1,27 @@
+version 1.0
+
+task cpu_platform {
+    input {
+        String cpu_platform
+    }
+    command {
+       apt-get install --assume-yes jq > /dev/null
+       NAME=`curl -s -H "Metadata-Flavor: Google" http://metadata.google.internal/computeMetadata/v1/instance/name`
+       ZONE=`basename \`curl -s -H "Metadata-Flavor: Google" http://metadata.google.internal/computeMetadata/v1/instance/zone\``
+       PROJECT=`curl -s -H "Metadata-Flavor: Google" http://metadata.google.internal/computeMetadata/v1/project/project-id`
+       curl -s -H "Authorization: Bearer `gcloud auth print-access-token`" "https://www.googleapis.com/compute/v1/projects/$PROJECT/zones/$ZONE/instances/$NAME?fields=cpuPlatform" | jq -r '.cpuPlatform'    }
+
+    runtime {
+        docker: "google/cloud-sdk:slim"
+        cpuPlatform: cpu_platform
+    }
+
+    output {
+        String cpuPlatform = read_string(stdout())
+    }
+}
+
+workflow cpu_platform {
```

Comment:
Let's rename this so that the task and workflow names don't collide... and I'm going to [have to make a bug ticket](https://github.com/broadinstitute/cromwell/issues/4705) to not allow this in WDL 1.0...